tools/rostopic/src/rostopic/__init__.py

```diff
@@ -210,7 +218,9 @@ def _rostopic_hz(topic, window_size=-1, filter_expr=None, use_wtime=False):
     while not rospy.is_shutdown():
         _sleep(1.0)
         rt.print_hz()
-
+        if pub:
+            rate, min_delta, max_delta, std_dev, n = rt.compute_hz()
+            pub.publish(rate)
```

Comment:
Would it make sense to publish more than just the rate? Maybe the user wants to plot the other values too.